src/test/java/com/ays/institution/controller/InstitutionControllerTest.java

```diff
@@ -0,0 +1,96 @@
+
+
+    private static final String BASE_PATH = "/api/v1/institutions";
+
+    @Test
+    void givenNothing_whenInstitutionStatusActive_thenReturnListInstitutionResponse() throws Exception {
+
+        String mockAccessToken = mockSuperAdminToken.getAccessToken();
+
+        Institution institution1 = new InstitutionBuilder().withValidFields().build();
+        Institution institution2 = new InstitutionBuilder().withValidFields().build();
+
+
+        List<Institution> mockInstitutions = Arrays.asList(
+                institution1,
+                institution2
+        );
+
+        // when
+        Mockito.when(institutionService.getInstitutionsSummary()).thenReturn(mockInstitutions);
+
+        // then
+        List<InstitutionResponse> mockInstitutionResponseList = institutionToInstitutionResponseMapper.map(mockInstitutions);
+        AysResponse<List<InstitutionResponse>> mockAysResponse = AysResponse.successOf(mockInstitutionResponseList);
+
+        mockMvc.perform(AysMockMvcRequestBuilders
+                        .get(BASE_PATH.concat("/summary"), mockAccessToken))
+                .andDo(MockMvcResultHandlers.print())
+                .andExpect(AysMockResultMatchersBuilders.status().isOk())
+                .andExpect(AysMockResultMatchersBuilders.time()
+                        .isNotEmpty())
+                .andExpect(AysMockResultMatchersBuilders.httpStatus()
+                        .value(mockAysResponse.getHttpStatus().getReasonPhrase()))
+                .andExpect(AysMockResultMatchersBuilders.isSuccess()
+                        .value(mockAysResponse.getIsSuccess()))
+                .andExpect(AysMockResultMatchersBuilders.response()
+                        .isNotEmpty());
+
+        // verify
+        Mockito.verify(institutionService, Mockito.times(1)).getInstitutionsSummary();
+
+    }
+
+
+    @Test
+    void givenNothing_whenUserUnauthorizedForSummary_thenReturnAccessDeniedException() throws Exception {
+
+        // Given
+        String mockAccessToken = mockUserToken.getAccessToken();
+
+
+        MockHttpServletRequestBuilder mockHttpServletRequestBuilder = AysMockMvcRequestBuilders
+                .get(BASE_PATH.concat("/summary"), mockAccessToken);
+
+        AysResponse<AysError> mockResponse = AysResponseBuilder.FORBIDDEN;
+        mockMvc.perform(mockHttpServletRequestBuilder)
+                .andDo(MockMvcResultHandlers.print())
+                .andExpect(AysMockResultMatchersBuilders.status().isForbidden())
+                .andExpect(AysMockResultMatchersBuilders.time().isNotEmpty())
+                .andExpect(AysMockResultMatchersBuilders.httpStatus().value(mockResponse.getHttpStatus().name()))
+                .andExpect(AysMockResultMatchersBuilders.isSuccess().value(mockResponse.getIsSuccess()))
+                .andExpect(AysMockResultMatchersBuilders.response().doesNotExist());
+
+    }
+
+}
```

Comment:
Bu sınıfın en altında bir tane fazladan boş satır olması gerekiyor. GitHub da bu konuda uyarıyor
![image](https://github.com/afet-yonetim-sistemi/ays-be/assets/61833677/3a2f6121-8df0-47ca-98b0-48e43d0f0d7f)

Reply:
Bu kısmı revise ettim